packages/alfa-rules/src/sia-r10/rule.ts

```diff
@@ -76,105 +77,116 @@ function hasTokens(input: string): boolean {
   return input.trim() !== "" && input.split(/\s+/).length > 0;
 }
 
+/**
+ * {@link https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofill-detail-tokens}
```

Comment:
```suggestion
 * {@link https://html.spec.whatwg.org/multipage/#autofill-detail-tokens}
```
(shortening URL a bit, and depending less on the page structure that might change)